pkg/controller/plan/validation.go

```diff
@@ -134,6 +135,11 @@ func (r *Reconciler) validate(plan *api.Plan) error {
 	if err != nil {
 		return err
 	}
+	// VDDK image
+	err = r.validateVddkImage(plan)
```

Comment:
as @liranr23 commented earlier, I would prefer to validate it using a webhook since we only check the validity of the configuration of the plan (in light of that of the provider) here